spec/features/use_nokogiri_to_seed_data_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+require 'open-uri'
+
+feature 'nokogiri creates stations', %(
+
+) do
+  scenario do
+    line_name = "Orange"
+    url = "http://www.mbta.com/schedules_and_maps/subway/lines/?route=#{line_name.upcase}"
+    doc = Nokogiri::HTML(open(url,
+      "User-Agent" => "Ruby/#{RUBY_VERSION}",
+      "From" => "foo@bar.invalid",
+      "Referer" => "http://www.ruby-lang.org/"))
+    station_names = []
+    doc.css('td a').each do |a|
+      if a.attributes["title"]
+        station_names << a.attributes["title"].value
+      end
+    end
+    station_names.map!{|name| name.sub(/ Station/,'')}
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.